sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -1321,5 +1322,56 @@ async def read_feed_ranges(
 
         feed_ranges = [FeedRangeInternalEpk(Range.PartitionKeyRangeToRange(partitionKeyRange)).to_dict()
                 for partitionKeyRange in partition_key_ranges]
-
         return (feed_range for feed_range in feed_ranges)
+
+    async def get_latest_session_token(self,
+                                       feed_ranges_to_session_tokens: List[Tuple[Dict[str, Any], str]],
+                                       target_feed_range: Dict[str, Any]
+                                       ) -> str:
+        """ **provisional** Gets the the most up to date session token from the list of session token and feed
```

Comment:
Maybe we could be a bit more generous with the warning here.
```suggestion
        """ **provisional** This method is still in preview and may be subject to breaking changes.
        
        Gets the the most up to date session token from the list of session token and feed
```

Reply:
Changed